java/fury-core/src/main/java/org/apache/fury/serializer/ObjectSerializer.java

```diff
@@ -228,6 +230,40 @@ public void write(MemoryBuffer buffer, T value) {
     writeContainerFields(buffer, value, fury, refResolver, classResolver);
   }
 
+  @Override
+  public T copy(T originObj) {
+    T newObj = newBean(constructor, type);
+    List<Field> fieldsList = getAllFieldCache();
+    for (Field field : fieldsList) {
+      try {
+        Object value = field.get(originObj);
+        field.set(newObj, fury.copy(value));
```

Comment:
This use reflection, which is pretty slow.
Could we use Plafform util here? Just like:

```java
for(FieldInfo info: fieldClassInfoList) {
long offset = info.offset;
swithc(info.classId) {
  ....
 case PRIMITIVE_INT: Platform.putInt(newObj,offset, Platform.getInt(originObj, info.offset); break;
 case PRIMITIVE_DOULE: Platform.putDouble(newObj, offset, Platform.getDouble(originObj, info.offset); break;
 case STRING, other immutable types ids:  Platform.putObject(newObj, offset, Platform.getObject(originObj, info.offset); break;
default: Platform.putObject(newObj, offset, fury.copy(Platform.getObject(originObj, info.offset))); break;
}
}
```

Reply:
No problem, I will try to use `Plafform util` to implement it and compare the execution efficiency with some common implementations